charts/cryostat/templates/deployment.yaml

```diff
@@ -190,6 +190,14 @@ spec:
           securityContext:
             {{- toYaml .Values.storage.securityContext | nindent 12 }}
         {{- if not .Values.minimal }}
+        - name: {{ printf "%s-%s" .Chart.Name "auth-proxy" }}
+          securityContext:
+            {{- toYaml .Values.authProxy.securityContext | nindent 12 }}
+          image: "{{ .Values.authProxy.image.repository }}:{{ .Values.authProxy.image.tag }}"
```

Comment:
It doesn't look like `authProxy.image.repository` and `authProxy.image.tag` are defined in `values.yaml`? Same with the `.securityContext` and `image.pullPolicy` etc.